baywatch.install

```diff
@@ -280,3 +284,53 @@ function baywatch_update_8008(&$sandbox) {
     $sandbox['#finished'] = $sandbox['#finished'] > 1 ? 1 : $sandbox['#finished'];
   }
 }
+
+/**
+ * Uninstall tide_authenticated_content.
+ * Except Vicpol and SSP.
+ */
+function baywatch_update_8009() {
+  $module_handler = \Drupal::moduleHandler();
+  $authenticated_module_exist = $module_handler->moduleExists('tide_authenticated_content');
+  // Load the site name out of configuration.
+  $config = \Drupal::config('system.site');
+  $site_name = $config->get('name');
+  if (($site_name !== 'Victoria Police' || $site_name !== 'Shared Service Provider Content Repository') && $authenticated_module_exist) {
+    \Drupal::service('module_installer')->uninstall(['tide_authenticated_content']);
+
+    // Remove permissions on tide_authenticated_content to Approver and Editor.
```

Comment:
I think line301 to line 336 should resides in hook_install in tide_authenticated_content module, and also remove the paragraph `user_authentication_block`

Reply:
just add an other PR https://github.com/dpc-sdp/tide_authenticated_content/pull/12